sktime/forecasting/base/tests/test_fh.py

```diff
@@ -264,3 +308,113 @@ def test_estimator_fh(freqstr):
     pred = forecaster.predict(np.arange(1, 27))
     expected_fh = ForecastingHorizon(np.arange(1, 27)).to_absolute(train.index[-1])
     assert_array_equal(pred.index.to_numpy(), expected_fh.to_numpy())
+
+
+# TODO: Replace this long running test with fast unit test
+def test_auto_ets():
+    """Fix bug in 1435.
+
+    https://github.com/alan-turing-institute/sktime/issues/1435#issue-1000175469
+    """
+    freq = "30T"
+    _y = np.arange(50) + np.random.rand(50) + np.sin(np.arange(50) / 4) * 10
+    t = pd.date_range("2021-09-19", periods=50, freq=freq)
+    y = pd.Series(_y, index=t)
+    y.index = y.index.to_period(freq=freq)
+    forecaster = AutoETS(sp=12, auto=True, n_jobs=-1)
+    forecaster.fit(y)
+    y_pred = forecaster.predict(fh=[1, 2, 3])
+    pd.testing.assert_index_equal(
+        y_pred.index,
+        pd.date_range("2021-09-19", periods=53, freq=freq)[-3:].to_period(freq=freq),
+    )
+
+
+# TODO: Replace this long running test with fast unit test
+def test_exponential_smoothing():
+    """Test bug in 1876.
+
+    https://github.com/alan-turing-institute/sktime/issues/1876#issue-1103752402.
+    """
+    y = load_airline()
+    # Change index to 10 min interval
+    freq = "10Min"
+    time_range = pd.date_range(
+        pd.to_datetime("2019-01-01 00:00"),
+        pd.to_datetime("2019-01-01 23:55"),
+        freq=freq,
+    )
+    # Period Index does not work
+    y.index = time_range.to_period()
+
+    forecaster = ExponentialSmoothing(trend="add", seasonal="multiplicative", sp=12)
+    forecaster.fit(y, fh=[1, 2, 3, 4, 5, 6])
+    y_pred = forecaster.predict()
+    pd.testing.assert_index_equal(
+        y_pred.index, pd.period_range("2019-01-02 00:00", periods=6, freq=freq)
+    )
+
+
+# TODO: Replace this long running test with fast unit test
+def test_theta_forecaster():
+    """Test example in notebook.
```

Comment:
the notebooks are already all tested as part of the CI/CD, i.e., that they all run without exception.

Reply:
I have added this test only for my convenience. But now that everything is fixed, I have removed it.